docs/docs/adding-analytics.md

```diff
@@ -26,24 +26,24 @@ You should now have a Tracking ID; take note of it, as your website will need to
 
 You can find this tracking ID later by going to `Admin > Tracking Info > Tracking Code`.
 
-## Using `gatsby-plugin-google-analytics`
+## Using `gatsby-plugin-gtag`
 
 Now, it's time to configure Gatsby to send page views to your Google Analytics account.
 
-We are going to use `gatsby-plugin-google-analytics`. For other analytics options (including Google Analytics gtag.js and Google Tag Manager), check [other Gatsby analytics plugins](#other-gatsby-analytics-plugins).
+We are going to use `gatsby-plugin-gtag`.
 
 ```shell
-npm install gatsby-plugin-google-analytics
+npm install --save gatsby-plugin-gtag
```

Comment:
You don't need the `--save` flag here (the plugin docs should probably also be updated, but that's a separate issue).

```suggestion
npm install gatsby-plugin-gtag
```